kong/conf_loader/init.lua

```diff
@@ -597,6 +597,8 @@ local CONF_PARSERS = {
   admin_gui_url = {typ = "string"},
   admin_gui_path = {typ = "string"},
   admin_gui_api_url = {typ = "string"},
+
+  cjson_encode_number_precision = { typ = "number", },
```

Comment:
```suggestion
  cjson_encode_number_precision = { typ = "number", default = 14 },
```

Reply:
all items in this table did not set `default`, so I follow this.